src/Glpi/Marketplace/View.php

```diff
@@ -268,6 +268,44 @@ public static function installed(
         self::displayList($plugins, "installed", $only_lis);
     }
 
+    /**
+     * Get the logo of a plugin
+     * Useful for custom plugins which will be used internally only and are not published in the internet
+     *
+     * @param array   $apidata  collected data of the plugin
+     * @param string  $key      plugins directory name
+     *
+     * @return string url to the plugins logo or empty string
+     */
+    public static function getLogo(array $apidata, string $key): string
+    {
+
+        // found a logo from online resource?
+        if (isset($apidata['logo_url']) && strpos(strtolower($apidata['logo_url']), '.png') !== false) return $apidata['logo_url'];
+
+        // absolute path to the plugin directory (for checking if a local file exists)
+        $pluginDir = GLPI_ROOT . '/plugins/' . $key . '/';
+
+        // relative path to the plugin directory (for html view)
+        $pluginPath = '../plugins/' . $key . '/';
```

Comment:
There is a method for that.

```suggestion
        // path to the plugin directory (for checking if a local file exists)
        $pluginDir = Plugin::getPhpDir($key);
```

Reply:
Thank you for your suggestion. I appreciate it.